src/main/java/org/opensearch/neuralsearch/processor/factory/RerankProcessorFactory.java

```diff
@@ -109,7 +133,8 @@ public static boolean shouldIncludeQueryContextFetcher(RerankType type) {
 
         /**
          * Create necessary queryContextFetchers for this processor
-         * @param config processor config object. Look for "context" field to find fetchers
+         *
+         * @param config                     processor config object. Look for "context" field to find fetchers
```

Comment:
Remove this extra space and revert to original form.

Reply:
Thanks, its back to how it was